src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerConstants.java

```diff
@@ -73,15 +73,15 @@ private CaseWorkerConstants() {
 
     public static final String ORIGIN_EXUI = "EXUI";
 
-    public static final String UP_FAILURE_ROLES = "can't modify roles for user in UP";
-
-    public static final String SUSPEND_USER_FAILED = "suspend user in UP failed";
+    public static final String UP_FAILURE_ROLES = "An update to the user is not possible at this moment."
+            + " Please try again later.";
 
     public static final String STATUS_ACTIVE = "ACTIVE";
 
     public static final String ALREADY_SUSPENDED_ERROR_MESSAGE = "user is suspended already";
 
-    public static final String UP_CREATION_FAILED = "Failed to create in UP with response status %s";
+    public static final String UP_CREATION_FAILED = "User creation is not possible at this moment."
```

Comment:
I think you need to remove the `String.format` in https://github.com/hmcts/rd-caseworker-ref-api/blob/RDCC-2376-error-message/src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java#L522

Reply:
Thanks for the catch Sujeeth. I have fixed it and awaiting the build.